### PR TITLE
added tests for invalid testset

### PR DESCRIPTION
**Pull request by enikao**
Depends on https://github.com/LIonWeb-org/lionweb-integration-testing/pull/4

### CODE REVIEW

**Comment by ftomassetti** [comment]
The build seems to fail

File: core/src/test/java/io/lionweb/lioncore/java/testset/Invalid.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        RuntimeException.class,
+        () -> {
+          List<Node> nodes =
+              JsonSerialization.getStandardSerialization().unserializeToNodes(path.toFile());
+          System.out.println(nodes);
+        });
+  }
+
+  private static Path findIntegrationTests() {
+    String propertyValue = System.getenv("integrationTestingDir");
```
Comment (ftomassetti) [inline core/src/test/java/io/lionweb/lioncore/java/testset/Invalid.java:52]: I am not sure how I can pass an environment variable when running the tests from within the IDE.
Perhaps I would instead either download the files as part of the test, or expect them to be in a certain directory relative to the root of the project (e.g., `build/integrationTests`)

**Comment by ftomassetti** [comment]
I will wait for https://github.com/LIonWeb-org/lionweb-integration-testing/pull/4 to be merged before completing the review. Does this make sense?

File: core/src/test/java/io/lionweb/lioncore/java/testset/Invalid.java (hunk 1 above)
Comment (enikao) [reply core/src/test/java/io/lionweb/lioncore/java/testset/Invalid.java:52]: Yes, I don't have a great idea for this either. Just implemented the next best thing besides hard-coded paths.

**Comment by ftomassetti** [comment]
Thank you Niko!
I have modified the configuration to create a separate source test for integration test and automatize the download of the integration repo. I now have 14 invalid files not recognized as invalid. I would merge this anyway, and fix those tests as soon as I can

**Pull request merged**